modules/reference/pages/internal-metrics-reference.adoc

```diff
@@ -2796,18 +2828,35 @@ The `namespace` label supports the following options for this metric:
 
 ---
 
+=== vectorized_ntp_archiver_missing
+
+Counts the number of missing offsets due to gaps. This metric is tracked on a per-topic and per-partition basis.
+
+*Type*: counter
+
+*Labels*:
+
+- `namespace`
+- `partition`
+- `shard`
+- `topic`
+
+---
+
 == vectorized_ntp_archiver_pending
 
-The difference between the last committed offset and the last offset uploaded to Tiered Storage for each partition. A value of zero for this metric indicates that all data for a partition is uploaded to Tiered Storage.
+Reports the difference between the last committed offset and the last offset uploaded to Tiered Storage for each partition. A value of zero for this metric indicates that all data for a partition is uploaded to Tiered Storage.
 
-This metric is impacted by the xref:reference:tunable-properties.adoc#cloud_storage_segment_max_upload_interval_sec[`cloud_storage_segment_max_upload_interval_sec`] tunable property. If this interval is set to 5 minutes, the archiver will upload committed segments to Tiered Storage every 5 minutes or less. If this metric continues growing for longer than the configured interval, it can indicate a potential network issue with the upload path for that partition.
+This metric is impacted by the xref:reference:tunable-properties.adoc#cloud_storage_segment_max_upload_interval_sec[`cloud_storage_segment_max_upload_interval_sec`] tunable property. If this interval is set to 5 minutes, the archiver uploads committed segments to Tiered Storage every 5 minutes or less. If this metric continues growing for longer than the configured interval, it can indicate a potential network issue with the upload path for that partition.
```

Comment:
```suggestion
This metric is impacted by the xref:reference:object-storage-properties.adoc#cloud_storage_segment_max_upload_interval_sec[`cloud_storage_segment_max_upload_interval_sec`] property. If this interval is set to 5 minutes, the archiver uploads committed segments to Tiered Storage every 5 minutes or less. If this metric continues growing for longer than the configured interval, it can indicate a potential network issue with the upload path for that partition.
```